contrib/builder/deb/generate.sh

```diff
@@ -73,6 +73,13 @@ for version in "${versions[@]}"; do
 		extraBuildTags+=' exclude_graphdriver_btrfs'
 	fi
 
+	# make backports the default for installing packages
+	if [ "$suite" = 'wheezy' ]; then
+		#   remove btrfs-tools so we can readd it and pull from backports
+		packages=( "${packages[@]/btrfs-tools}" )
+		packages+=('--no-install-recommends && apt-get install -y -t wheezy-backports btrfs-tools')
```

Comment:
Although from what I can find, it does appear that this restriction is "good enough" still (ie, 3.8 appears to contain the header fields we require).